src/types/modules/react-native-web-linear-gradient.d.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/consistent-type-definitions */
+declare module 'react-native-web-linear-gradient' {
```

Comment:
Types should be the same as in `react-native-linear-gradient` library. Looking at the `react-native-web-linear-gradient` [code](https://github.com/react-native-web-community/react-native-web-linear-gradient/blob/master/src/index.js).

```ts
declare module 'react-native-web-linear-gradient' {
    import LinearGradient from 'react-native-linear-gradient';

    export default LinearGradient;
}
```